### PR TITLE
Docker Container Quickstart Docs

### PR DESCRIPTION
**Pull request by sdreyer**
# Description

A quickstart guide on how to use featureform with the standalone container

### Select type(s) of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update

# Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have fixed any merge conflicts

### CODE REVIEW

File: docs/quickstart-docker.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+---
+description: A quick start guide for Featureform with Docker.
+---
+
+# Quickstart (Docker)
+This quickstart uses the standalone containered version of Featureform. It can be used to connect to the same providers 
+as the Kubernetes Hosted version, but lacks the scaling capabilities.
+
+This quickstart will walk through creating a few simple features, labels, and a training from a fraud
```
Comment (Sami1309) [inline docs/quickstart-docker.md:9]: *training set

File: docs/quickstart-docker.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+- Python 3.7+
+- Docker
+
+## Step 1: Install The Featureform CLI
+```shell
+pip install featureform
+```
+
+## Step 2: Pull Containers
+We'll use postgres and redis as providers in this quickstart. We've created a Featureform Postgres container that
```
Comment (Sami1309) [inline docs/quickstart-docker.md:23]: should postgres and redis be capitalized

File: docs/quickstart-docker.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+docker run -d -p 6379:6379 redis
+docker run -p 7878:7878 -p 80:80 featureformcom/featureform
+```
+
+Note: If using an M1 Mac, run the following command instead for compatibility 
+```shell
+docker run -p 7878:7878 -p 80:80 -e ETCD_ARCH="ETCD_UNSUPPORTED_ARCH=arm64" featureformcom/featureform
+```
+
+Wait 30-60 seconds to allow the Postgres container to fully initialize the sample data.
```
Comment (Sami1309) [inline docs/quickstart-docker.md:40]: should there be a check here to run the get or list to check that the data is ready

File: docs/quickstart-docker.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+Featureform definitions can be stored as both a local file and URL. Multiple files can be applied at the same time. 
+
+We'll set the `--insecure` flag since we're using an unencrypted endpoint on the container.
+
+```shell
+featureform apply https://featureform-demo-files.s3.amazonaws.com/definitions.py --insecure
+```
+
+## Step 5: Dashboard and Serving
+The dashboard is available at [localhost](http://localhost)
```
Comment (Sami1309) [inline docs/quickstart-docker.md:59]: does just localhost default to 80?

**Review by Sami1309 — COMMENTED** [review]
looks very good, just very minor comments

File: docs/quickstart-docker.md (hunk 1 above)
Comment (sdreyer) [reply docs/quickstart-docker.md]: fixed

File: docs/quickstart-docker.md (hunk 2 above)
Comment (sdreyer) [reply docs/quickstart-docker.md]: fixed

File: docs/quickstart-docker.md (hunk 3 above)
Comment (sdreyer) [reply docs/quickstart-docker.md:40]: This is postgres itself that needs to initialize. Kind of annoying but no great way to check at the moment

File: docs/quickstart-docker.md (hunk 4 above)
Comment (sdreyer) [reply docs/quickstart-docker.md:59]: yeah

**Review by Sami1309 — APPROVED** [review]
minor errors but otherwise lgtm

File: docs/quickstart-docker.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+    "fraud_training",
+    label="fraudulent",
+    features=["avg_transactions"],
+)
+```
+{% endcode %}
+
+## Serving
+
+We can serve single features from Redis with the Serving Client. The Feature method takes the name of the feature
```
[the remark this answers is not in the document]
Comment (sdreyer) [reply docs/quickstart-docker.md]: Just addressed it by its exact name

**Pull request merged**